k3s-with-registry.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+#
+# Starts a k3s cluster (via k3d) with local image registry enabled,
```

Comment:
unsure if this script needs a copyright statement like [the other one](https://github.com/windmilleng/kind-local/blob/master/kind-with-registry.sh#L3)?

Reply:
did you copy it from their repo? what do the copyright license terms say?

Reply:
No, I modified our kind script which modified Kind's kind script. But on further reflection, very little is the same between this script and Kind's kind-with-registry script, so it's probably fine...